playbooks/roles/ufw/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Ensure UFW allows SSH (22/TCP)
+  ufw:
+    rule="allow"
+    name="OpenSSH"
+
+- name: Ensure UFW is enabled and denies by default
+  ufw:
+    state="enabled"
+    policy="deny"
+    direction="incoming"
+
+- name: Ensure UFW allows L2TP / IPSec (500/UDP)
```

Comment:
What do you think about combining the L2TP/IPsec firewall rules into a single task and looping over the ports using `with_items`?